packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/resources/dataservice/ProjectResource.java

```diff
@@ -37,7 +36,7 @@ public Response getProjects(
 
 		projects.forEach(project -> {
 			Assets assets = proxy.getAssets(project.getProjectID(), Arrays.asList("datasets", "models", "publications"));
-			Map metadata = new HashMap();
+			Map<String, String> metadata = new HashMap<>();
```

Comment:
Raw types shouldn't have been used here.  This is unrelated to this PR at all but was like nails on a chalkboard to me ;)

Reply:
Make sense, thanks!